packages/react-composites/review/react-composites.api.md

```diff
@@ -290,16 +311,34 @@ export type CallIdChangedListener = (event: {
 export type CallIdentifierKinds = CommunicationUserKind | PhoneNumberKind | MicrosoftTeamsUserKind | UnknownIdentifierKind;
 
 // @public (undocumented)
-export interface ChatAdapter {
-    clearErrors(errorTypes: ErrorType[]): void;
-    // (undocumented)
-    dispose(): void;
+export interface ChatAdapter extends ChatAdapterHandlers, AdapterState<ChatState>, AdapterDisposal, AdapterErrorHandlers, ChatAdapterSubscribers {
+}
+
+// @public
+export type ChatAdapterErrors = {
+    [operation: string]: Error;
+};
+
+// @public (undocumented)
+export interface ChatAdapterHandlers {
```

Comment:
Similar to above, this could become `ChatAdapterThreadManagement`

Reply:
updated